applications/api-cat/src/main/java/no/acat/spec/converters/SwaggerJsonSpecConverter.java

```diff
@@ -19,7 +20,10 @@ public static boolean canConvert(String spec) {
 
     public static OpenAPI convert(String spec) throws ParseException {
         try {
-            return new SwaggerConverter().readContents(spec, null, null).getOpenAPI();
+            io.swagger.v3.oas.models.OpenAPI parseResult = new SwaggerConverter().readContents(spec, null, null).getOpenAPI();
+            OpenAPI result = new OpenAPI();
+            BeanUtils.copyProperties(parseResult, result);
```

Comment:
Det er sikkert en fornuftig grunn, men hvorfor kopierer du resultatet med BeanUtils i stedet for å bare returnere resultatet direkte ?